e2e/nextjs/pages-router/app/package.json

```diff
@@ -10,7 +10,7 @@
     },
     "dependencies": {
         "@devcycle/nextjs-sdk": "file:../../../../dist/sdk/nextjs",
-        "next": "14.0.5-canary.38",
+        "next": "14.1.1",
```

Comment:
@ajwootto also a reason we are on a different canary release here?

Reply:
no not really. Also the latest next version is 14.2